en/docs/learn/x509certificate-authenticator.md

```diff
@@ -0,0 +1,703 @@
+
+### Using an identity claim for the X509 certificate or working with read only user stores
+
+If you are using an identity claim to store X509 Certificates, e.g.,
+`                   http://wso2.org/claims/identity                  ,        `
+or if you are working with a read-only user store, the certificate gets
+stored in the `         DATA_VALUE        ` column of the
+`         IDN_IDENTITY_USER_DATA        ` table. The default DB script
+sets the column size to 255 characters but in this case, the certificate
+value has more than 255 characters. Therefore, you need to update the
+column size to a higher value.
+
+Follow the steps given below to update the column size:
+
+You do not need to update the column size if you are using WSO2 IS 5.4.0
+or above 5.4.0.
+
+-   Refer this
+    [link](https://docs.wso2.com/display/ADMIN44x/Browsing+the+H2+Database)
+    to browse the H2 database of WSO2 products, and execute the query
+    given below to alter the column size of the H2 database.
+
+    ``` java
+    ALTER TABLE IDN_IDENTITY_USER_DATA ALTER DATA_VALUE VARCHAR(2048);
+    ```
+
+-   Refer the table given below to find out the queries you can use for
+    the databases listed below:
+
+    <table>
+    <thead>
+    <tr class="header">
+    <th>Database</th>
+    <th>Query to alter the column</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr class="odd">
+    <td>MySQL</td>
+    <td><code>               ALTER TABLE IDN_IDENTITY_USER_DATA CHANGE DATA_VALUE DECIMAL VARCHAR(2048)              </code></td>
+    </tr>
+    <tr class="even">
+    <td>Oracle</td>
+    <td><code>               IDN_IDENTITY_USER_DATA modify (DATA_VALUE varchar(2048));              </code></td>
+    </tr>
+    <tr class="odd">
+    <td>MSSQL</td>
+    <td><code>               ALTER TABLE IDN_IDENTITY_USER_DATA              </code><br />
+    <code>               ALTER COLUMN DATA_VALUE VARCHAR(2048) NOT NULL;              </code></td>
+    </tr>
+    <tr class="even">
+    <td>PostgreSQL</td>
+    <td><code>               ALTER TABLE IDN_IDENTITY_USER_DATA ALTER COLUMN DATA_VALUE TYPE VARCHAR(2048); Configuring the X509 Certificate for the app              </code></td>
+    </tr>
+    </tbody>
+    </table>
+
+### Using a wso2 claim for the X509 certificate
+
+If you use are using a wso2 claim to store X509 Certificate, e.g.,
+`                   http://wso2.org/claims                 ` , the
+certificate gets stored as a user attribute in the
+`         UM_ATTR_VALUE        ` column of the
+`         UM_USER_ATTRIBUTE        ` table. The default DB script sets
+the column size to 1024 characters but in this case, the certificate
+value is having more than 1024 characters Therefore, you need to update
+the column size to a higher value.
+
+Follow the steps given below to update the column size:
+
+You do not need to update the column size if you are using WSO2 IS 5.4.0
+or above 5.4.0.
```

Comment:
According to this statement, for 5.10 we don't need to follow these steps so shall we remove this entire section ?